subprojects/common/time/src/main/kotlin/com/avito/time/DefaultTimeProvider.kt

```diff
@@ -1,11 +1,21 @@
 package com.avito.time
 
+import java.text.DateFormat
+import java.text.SimpleDateFormat
 import java.util.Calendar
 import java.util.Date
+import java.util.Locale
+import java.util.TimeZone
 import java.util.concurrent.TimeUnit
 
+/**
+ * Used in android runtime < 26, so no java.time API yet
+ * consider using ThreeTenABP
+ */
 class DefaultTimeProvider : TimeProvider {
 
+    override val timeZone: TimeZone = TimeZone.getTimeZone("Europe/Moscow")
```

Comment:
Why not default timezone?

Reply:
yeah, you're right, let me check how a default timezone resolved first